package.json

```diff
@@ -12,6 +12,8 @@
     "debug": "*",
     "download": "0.1.7",
     "ecstatic": "0.5.4",
+    "eslint": "0.21.0",
+    "eslint-plugin-no-unsafe-innerhtml": "0.1.2",
```

Comment:
I'm not exactly sure how npm modules work these days -- but I think we're still using https://github.com/mozilla-b2g/gaia-node-modules/ for now. So you'll need to add eslint + the plugin there too, follow the instructions that are in the README, and update the commit hash in `gaia_node_modules.revision`.